drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/DefaultValueWidgetFactory.java

```diff
@@ -85,12 +86,8 @@ public static Widget getDefaultValueWidget(final AttributeCol52 ac,
                                                final boolean isReadOnly,
                                                final DefaultValueChangedEventHandler defaultValueChangedEventHandler) {
         Widget editor = null;
-        final String attributeName = ac.getAttribute();
-        if (attributeName.equals(RuleAttributeWidget.RULEFLOW_GROUP_ATTR)
-                || attributeName.equals(RuleAttributeWidget.AGENDA_GROUP_ATTR)
-                || attributeName.equals(RuleAttributeWidget.ACTIVATION_GROUP_ATTR)
-                || attributeName.equals(RuleAttributeWidget.TIMER_ATTR)
-                || attributeName.equals(RuleAttributeWidget.CALENDARS_ATTR)) {
+        final String attribute = ac.getAttribute();
+        if (Objects.equals(Attribute.getAttributeDataType(attribute), DataType.TYPE_STRING)) {
```

Comment:
@jomarko I don't know the functionality involved, but I see a logic change, here. With these changes, you will manage an additional CASE not managed before:     `DIALECT("dialect", DataType.TYPE_STRING)`. Is this intended? Most probably it is, let me know :)

Reply:
@yesamer very good point, to be sure, I reorder the of blocks, first I will check for specific attributes, then just according data type. thank you